### ISSUE TITLE
Nolok's gum shield is weaker than other gum shields

### ISSUE THREAD

**Issue by Alayan-stk-2** [supertuxkart/stk-code#4940]
For all the karts but Nolok, when using a new gum shield while already having one on, the remaining time is added to the full time of a new shield. With Nolok, it is not, so only the normal time of a new gum shield is applied.

The impact is quite minor in most situations.

Do not fix it in master, it is fixed in the 2.0 balance branch that will be merged later.

**Comment by Wuzzy2** [comment]
If I would take a guess, this sounds like this was a tweak specifically made for the final challenge in Story Mode and multiplayer simply wasn't considered when this was done.

**Comment by Lulu-Games** [comment]
> If I would take a guess, this sounds like this was a tweak specifically made for the final challenge in Story Mode and multiplayer simply wasn't considered when this was done.

That sounds possible, but I don't think, that that's a minor issue, because it's unfair for the other players! Why isn't there the possibility to add custom shields for each player? That would be cool, but I don't think, that this is easy to implement.

**Comment by Wuzzy2** [comment]
I agree this is not a minor issue and it was not meant as an excuse. In multiplayer, bubblegum shield should have equal strength for everyone. But the Story Mode Nolok battle at Fort Magma could keep the special shield tweak just fine.

But sounds like this bug is alredy fixed, it's basically just waiting for the 2.0 release, apparently.

**Comment by Lulu-Games** [comment]
In the Story mode that really makes sense

**Comment by Alayan-stk-2** [comment]
Fixed in the Evolution branch as a minor tweak within https://github.com/supertuxkart/stk-code/commit/6224d35fff261f0d4bcfa3a7f020f965abde1d4e
